daemon/conn/conn.go

```diff
@@ -117,13 +149,13 @@ func GetAWSConfigSession(cn connAttr, c *cfg.Config, roleArn string, region stri
 		es := getDefaultSession()
 		awsRegion, err = cn.getEC2Region(es)
 		if err != nil {
-			log.Errorf("Unable to retrieve the region from the EC2 instance %v\n", err)
+			awsRegion = getRegionFromECSMetadata()
 		} else {
 			log.Debugf("Fetch region %v from ec2 metadata", awsRegion)
 		}
```

Comment:
I think that's a good call, due to the timeout when EC2 metadata is unavailable. I did not, however put logging there, only because there is already such logging in getRegionFromECSMetadata()